File: ce/snippets/visualbasic/CRMV8/queryvb/vb/uselinqwithdotnetdataservicesde2.vb
' <snippetuselinqwithdotnetdataservicesde2>


 Dim pageSize As Integer = 5

 Dim accountsByPage = From a In orgContext.CreateQuery("account") _
                      Select a("name")
 Console.WriteLine("Skip 10 accounts, then Take 5 accounts")
 Console.WriteLine("======================================")
 For Each name In accountsByPage.Skip(2 * pageSize).Take(pageSize)
  Console.WriteLine(name)
 Next name

' </snippetuselinqwithdotnetdataservicesde2>

File: ce/customer-engagement/on-premises/snippets/csharp/CRMV8/query/cs/uselinqwithdotnetdataservicesde2.cs
// <snippetuselinqwithdotnetdataservicesde2>


 int pageSize = 5;

 var accountsByPage = from a in orgContext.CreateQuery("account")
                      select a["name"];
 System.Console.WriteLine("Skip 10 accounts, then Take 5 accounts");
 System.Console.WriteLine("======================================");
 foreach (var name in accountsByPage.Skip(2 * pageSize).Take(pageSize))
 {
  System.Console.WriteLine(name);
 }

// </snippetuselinqwithdotnetdataservicesde2>